iOSClient/Data/NCManageDatabase+Metadata.swift

```diff
@@ -61,8 +61,8 @@ class tableMetadata: Object, NCUserBaseUrl {
     @objc dynamic var iconName = ""
     @objc dynamic var iconUrl = ""
     @objc dynamic var isExtractFile: Bool = false
-    @objc dynamic var livePhoto: Bool = false
     @objc dynamic var livePhotoFile = ""
+    @objc dynamic var livePhotoServer: Bool = false
```

Comment:
```suggestion
    /// A flag indicating if the file is sent as a live photo from the server, or if we should detect it as such and convert it client-side. 
    /// Usually this will be false if it's an old media and there is no `<metadata-files-live-photo xmlns="[http://nextcloud.org/ns\"/>]` flag in the metadata, and true if there is.
    @objc dynamic var isFlaggedAsLivePhotoByServer: Bool = false
```